doc/rst/source/makecpt.rst

```diff
@@ -328,6 +334,10 @@ To instead add unique category labels A, B, C, ... to a 10-item categorical CPT,
 
     gmt makecpt -Cjet -T0/10/1 -F+cA
 
+To make a categorical CPT with string keys instead of numerical lookup values, try::
+
+    gmt makecpt -Ccategorical -Twood,water,gold 
```

Comment:
```suggestion
    gmt makecpt -Ccategorical -F+kWood,Water,Gold
```

Reply:
Yes, had a change of heart after first commit and @joa-quim and I decided we might as well let -T take the keys if a comma-separated list.  We use +k if we want to read a file or auto-sequence the keys.